hw12-solid/src/main/java/ru/otus/homework/ATM.java

```diff
@@ -0,0 +1,15 @@
+package ru.otus.homework;
+
+import java.util.ArrayList;
+
+public interface ATM {
+    void putBanknote(Banknote banknote);
+
+    void putBanknotes(ArrayList<Banknote> banknotes);
+
+    void printBalance();
+
+    int getBalance();
+
+    ArrayList<Banknote> getSum(long sum);
```

Comment:
```suggestion
    List<Banknote> getSum(long sum);
```

Программируйте на уровне интерфейсов.